data/compliance_profiles/CentOS/6/nist_800_53_rev4.json

```diff
@@ -2107,7 +2107,7 @@
         "value": {
           "programs": [
             "sudo",
-            "sudosh",
+            "tlog",
             "yum",
             "audispd",
             "auditd",
```

Comment:
Did you want to order these in the new way they were ordered in simp_rsyslog?  Would make maintenance easier